### ISSUE TITLE
Trap-door Spider

### ISSUE THREAD

**Issue by vadis365** [ganymedes01/TheErebus#640]
Found in the Petrified forest biome.
Trap-door Spider hides in 1 deep holes with a 'dust layer' block covering. Ambushes players that come within a 2 block range. Grabs the player and inflicts damage and/or poison. Will not leave it's hole and will hide when a player is out of range. Immune to projectile attacks. Usual spider drops I guess (unless someone comes up with something good).

**Comment by radical-radiodont** [comment]
Maybe make it drop something used for making pitfall traps?

**Comment by vadis365** [comment]
Yeah, some pitfall traps would be fun - could make some bamboo spikes and stuff and other trap related things :)
